src/components/Sentence/index.js

```diff
@@ -0,0 +1,80 @@
+
+// This component is built with:
+// React: https://facebook.github.io/react/
+
+import React from 'react';
+
+const Sentence = React.createClass({
+  adjectives: [
+    'leathery',
+    'salmony',
+    'beknighted',
+    'scintillating',
+    'flabbergasted',
+    'distracted',
+    'shattered',
+    'foamy',
+    'primitive',
+    'blathery',
+    'crumpled',
+    'tiny'
+  ],
+  nounsSubject: [
+    'handkerchief',
+    'elephant tusk',
+    'umbrella',
+    'big foot',
+  ],
+  nounsObject: [
+    'fancily decorated toothpick',
+    'bunch of blue balloons',
+    'washing machine lint filter with towel fluff in it',
+    'banana with a brown spot'
+
+
+  ],
+  verbs: [
+    'skitter',
+    'froth',
+    'zoom forward',
+    'stew',
+    'waggle its little finger',
+    'stomp',
+    'chew a purple gumball',
+    'fray'
+  ],
+  // adverbs: [
+  //   'speedily',
+  //   'clammily',
+  //   'frighteningly',
+  //   'knowingly',
+  //   'blisteringly',
+  //   'alarmingly'
+  // ],
+  pronouns: [
+    'am I',
+    'art thou',
+    'is thee',
+    'are we',
+    'is he',
+    'is she',
+    'is it'
+  ],
+  possessives: [
+    'my',
+    'your',
+    'their',
+    'his',
+    'her',
+    'our'
+  ],
+  getWord: function(poSp){
```

Comment:
Don't be afraid of long function or variable names. Clarity/readability/understandability come before brevity or cleverness. In my codebase at work, we have a function named `setDesiredSuspensionRestartDateAndValidateDates`. It's very clear what it does, right? Imagine if instead it said `setSusResDateVal`!